index.js

```diff
@@ -2292,6 +2292,24 @@
                R.pipe(R[name], Just, R.filter(R.gte(_, 0))));
   };
 
+  //# append :: a -> Array a -> Array a
```

Comment:
wow i'm close to giving up x__X  - The one huge file is driving me bonkers. I don't think I've ever needed so many attempts to get something so trivial done. Sorry.